### PR TITLE
Adding post for study group framework

### PR DESCRIPTION
**Pull request by kquerna-godaddy**
### Landing Page
Screen Shot 2022-05-27 at 11 20 42 AM

### List View
Screen Shot 2022-05-27 at 11 20 13 AM

### Blog Post 
Screen Shot 2022-05-27 at 11 20 23 AM

### CODE REVIEW

File: _posts/2022-05-27-study-group-framework.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+title: Life Getting in the Way of Learning? Use This Study Group Framework
+date: 2022-05-27 00:00:00 -0700
+cover: /assets/images/study-group-framework/cover.jpeg
+options:
+  - full-bleed-cover
+excerpt: How to keep learning new skills while meeting deadlines at work and maintaining a good work-life balance.
+keywords: skill development, certification, procrastination, conflicting priorities
+authors:
+  - name: Rebekah Gil
+    title: Mgr - Software Develoment
```
Comment (kbader-godaddy) [inline _posts/2022-05-27-study-group-framework.md:12]:
```suggestion
    title: Mgr - Software Development
```

File: _posts/2022-05-27-study-group-framework.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    photo: /assets/images/rgil.jpeg
+  - name: Mayur Jain
+    title: Director of Engineering
+    url: https://www.linkedin.com/in/jainmayur/
+    photo: /assets/images/mjain.jpeg
+---
+
+## Life Getting in the Way of Learning? Use This Study Group Framework
+
+I started leading the Cloud Platforms Services team at GoDaddy in late 2019. My team’s charter was to develop and maintain GoDaddy’s home grown AWS Service Catalog products. Additionally, my team provided guidance and support to engineers when they ran into issues deploying infrastructure in AWS. We were one of a handful of teams tasked with ensuring a successful adoption of the AWS Cloud by GoDaddy engineers. At the same time we were also trying to shift the company culture towards Developer Driven [DevSecFinOps](https://www.godaddy.com/engineering/2022/01/28/raising-the-bar-for-devsecops-beyond/).
```
Comment (kbader-godaddy) [inline _posts/2022-05-27-study-group-framework.md:23]:
```suggestion
I started leading the Cloud Platforms Services team at GoDaddy in late 2019. My team’s charter was to develop and maintain GoDaddy’s homegrown AWS Service Catalog products. Additionally, my team provided guidance and support to engineers when they ran into issues deploying infrastructure in AWS. We were one of a handful of teams tasked with ensuring the successful adoption of the AWS Cloud by GoDaddy engineers. At the same time, we were also trying to shift the company culture towards Developer Driven [DevSecFinOps](https://www.godaddy.com/engineering/2022/01/28/raising-the-bar-for-devsecops-beyond/).
```

File: _posts/2022-05-27-study-group-framework.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+As if dealing with support requests from engineers everyday wasn't enough, my seven-member team was also responsible for keeping up with the fast pace of innovation at AWS. To top it off, most of us only had a superficial knowledge of AWS services when we started. It was like flying an airplane not knowing what a lot of the knobs did. Sound familiar?
```
Comment (kbader-godaddy) [inline _posts/2022-05-27-study-group-framework.md:25]:
```suggestion
As if dealing with support requests from engineers every day wasn't enough, my seven-member team was also responsible for keeping up with the fast pace of innovation at AWS. To top it off, most of us only had a superficial knowledge of AWS services when we started. It was like flying an airplane, not knowing what a lot of the knobs did. Sound familiar?
```

**Comment by vylawrence-godaddy** [comment]
Looks great!

File: _posts/2022-05-27-study-group-framework.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+### Conflicting Priorities?
+
+Keeping up with the pace of innovation in technology while delivering on your commitments and maintaining a healthy work-life balance can sound like a pipe dream to many people. It sure sounded like one to me!
+
+In the spring of 2020, I wanted to earn the AWS Solutions Architect Associate certification. I found some motivation, purchased an online prep-course (thankfully, GoDaddy provided a monthly learning allowance), and started studying. And then…life happened. An urgent deadline at work, daughter’s school project, extra social commitments, yada yada yada and I had to put the studying on the back burner.
```
Comment (kbader-godaddy) [inline _posts/2022-05-27-study-group-framework.md:31]:
```suggestion
In the spring of 2020, I wanted to earn the AWS Solutions Architect Associate certification. I found some motivation, purchased an online prep course (thankfully, GoDaddy provided a monthly learning allowance), and started studying. And then…life happened. An urgent deadline at work, my daughter’s school project, extra social commitments, yadda yadda yadda and I had to put the studying on the back burner.
```

File: _posts/2022-05-27-study-group-framework.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+### Am I the only one?
+
+I tried to block out time to prepare for the exam but failed again. Then I realized that I must not be the only one facing this issue. So I discussed my struggles with one of my direct reports, Rebekah Gil, and she mentioned she had struggled with the same problem. That’s when the idea of forming a study group - to keep each other accountable - came to mind. I ran the idea by her, and she was on-board! I had found my partner-in-crime.
```
Comment (kbader-godaddy) [inline _posts/2022-05-27-study-group-framework.md:35]:
```suggestion
I tried to block out time to prepare for the exam but failed again. Then I realized that I must not be the only one facing this issue. So I discussed my struggles with one of my direct reports, Rebekah Gil, and she mentioned she had struggled with the same problem. That’s when the idea of forming a study group - to keep each other accountable - came to mind. I ran the idea by her, and she was on board! I had found my partner-in-crime.
```

**Review by vylawrence-godaddy — APPROVED** [review]

File: _posts/2022-05-27-study-group-framework.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+### Time to do some planning
+
+The two of us joined forces to choose an online prep-course. It had 10 chapters and roughly 15 hours of video content. We divided the course into five parts with roughly three hours of content to watch every week. This gave us five weeks to complete the course and we added a sixth week to the schedule to take practice quizzes. We also enrolled a few engineers as mentors who were much more knowledgeable than us on AWS services and had taken the certification exam already. We assigned one mentor for each week and gave them the task of coming up with quizzes on the topics for that week. We also asked them to attend our weekly meetings to answer any questions the participants had.
```
Comment (kbader-godaddy) [inline _posts/2022-05-27-study-group-framework.md:39]:
```suggestion
The two of us joined forces to choose an online prep course. It had 10 chapters and roughly 15 hours of video content. We divided the course into five parts with roughly three hours of content to watch every week. This gave us five weeks to complete the course and we added a sixth week to the schedule to take practice quizzes. We also enrolled a few engineers as mentors who were much more knowledgeable than us on AWS services and had taken the certification exam already. We assigned one mentor for each week and gave them the task of coming up with quizzes on the topics for that week. We also asked them to attend our weekly meetings to answer any questions the participants had.
```

**Review by kbader-godaddy — APPROVED** [review]

**Pull request merged**